nexus/caller/app/workflows.go

```diff
@@ -0,0 +1,43 @@
+package app
+
+import (
+	"github.com/temporalio/samples-go/nexus/service"
+	"go.temporal.io/sdk/workflow"
+)
+
+const (
+	TaskQueue    = "my-caller-workflow-task-queue"
+	endpointName = "my_nexus_endpoint_name"
+)
+
+func EchoCallerWorkflow(ctx workflow.Context, message string) (string, error) {
```

Comment:
Other samples just put the workflow in the samples root folder, don't think we need to do anything different here.

https://github.com/temporalio/samples-go/blob/main/helloworld/helloworld.go